src/nvim/marktree.c

```diff
@@ -905,7 +905,6 @@ bool marktree_splice(MarkTree *b,
           refkey(b, itr->node, itr->i);
           refkey(b, enditr->node, enditr->i);
         } else {
-          past_right = true;
```

Comment:
This assignment was informative even if it there wasn't being currently used :(

Reply:
Oh, sorry. Can I ignore clang analyze by sonething like annotation comment?

Reply:
(void)past_right;